sycl/tools/sycl-trace/CMakeLists.txt

```diff
@@ -60,28 +60,48 @@ add_dependencies(sycl_pi_trace_collector pi-pretty-printers ze-pretty-printers)
 
 if(SYCL_BUILD_PI_CUDA)
 
+  find_package(CUDA 10.1 REQUIRED)
   target_compile_definitions(sycl_pi_trace_collector
     PRIVATE
       $<$<BOOL:${SYCL_BUILD_PI_CUDA}>:USE_PI_CUDA>
   )
 
-  find_package(CUDAToolkit 10.1 REQUIRED)
+  # The following find_path can be removed when FindCUDA -> FindCUDAToolkit, and
+  # _CUPTI_INCLUDE_DIR -> CUDAToolkit_CUPTI_INCLUDE_DIR
+  find_path(_CUPTI_INCLUDE_DIR cupti.h PATHS
+      "${CUDA_TOOLKIT_ROOT_DIR}/extras/CUPTI/include"
+      "${CUDA_INCLUDE_DIRS}/../extras/CUPTI/include"
+      "${CUDA_INCLUDE_DIRS}"
+      NO_DEFAULT_PATH)
+
```

Comment:
I suggest we move the code shared between `sycl/tools/sycl-trace/CMakeLists.txt` and `sycl/plugins/cuda/CMakeLists.txt`
to https://github.com/intel/llvm/tree/sycl/sycl/cmake/modules to avoid duplication.

Reply:
Good point! I refactored the CMake file in the following commit.